src/components/Pagination.jsx

```diff
@@ -15,12 +16,12 @@ export const Pagination = ({
   listOfItems,
   setCurrentPageItems,
 }) => {
-  const [activePageIDState, setActivePageIDState] = useState(1);
+  const [currentPageState, setCurrentPageState] = useState(1);
```

Comment:
It's not necessary (or recommended even) to name your state variables with `State` keyword suffix. `currentPage` tells you enough what it does :)